### PR TITLE
Fix viewport

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/1163321984198618/1205404061469342/f 

### Description
This PR is a first attempt to fix our viewport issues

### Steps to test this PR

- [x] Install the app, go to serp and when you see the onboarding dialog select to hide tips forever
- [x] Go to twitter.com
- [x] You should see the whole cookies popup at the bottom of the screen and all of it should be visible.
- [x] Scrolling up and down should hide and show the URL as expected and the cookies popup should (more or less) stay in place (it will jump a bit).
- [x] On the same tab go to `reddit.com`, a container should be shown at the bottom of the screen and all of it should be visible. Scroll up and down and it should still be there.
- [x] Open a new tab and go to `reddit.com`, a container should be shown at the bottom of the screen and all of it should be visible. Scroll up and down and it should still be there.
- [x] On the same tab go to `twitter.com` and you should see the whole cookies popup at the bottom of the screen and all of it should be visible.

### CODE REVIEW

**Comment by marcosholgado** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3716** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3716" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3716?utm_source=stack-comment).

**Review by joshliebe — APPROVED** [review]
lgtm 👍

A bit jumpy but otherwise seems to work well. 

I'd like to run this on our nightlies for a while just to make sure it doesn't cause any adverse affects before releasing, so I've added a "do not merge" tag.

We can merge after the next release has gone out.

- [ ] Release 5.176.0

**Pull request merged**
